src/components/common/SearchBar/index.tsx

```diff
@@ -0,0 +1,53 @@
+'use client';
+
+import { ChangeEvent, InputHTMLAttributes, useState } from 'react';
+
+import SearchIcon from '/public/SearchIcon.svg';
+import DeleteIcon from '/public/DeleteIcon.svg';
+
+import Input from '../Input';
+
+interface SearchBarProps extends InputHTMLAttributes<HTMLInputElement> {
+  icon?: React.ReactNode;
+}
+
+export default function SearchBar({ icon }: SearchBarProps) {
+  const [inputValue, setInputValue] = useState('');
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setInputValue(e.target.value);
+  };
+
+  const clearInput = () => {
+    setInputValue('');
+  };
```

Comment:
요 로직은 제가 작성해둔 `useInput`을 사용하시면 편하실 것 같습니다!

```suggestion
  const [text, onTextChange, resetText]= useInput('');
  
  // 중략
  
          <Input
          value={text}
          onChange={onTextChange}
        />
```

Reply:
https://github.com/depromeet/ddoeat_client/pull/37/commits/a727b8ef12febfa608797d834abc25b7ff6d3793 에서 리뷰 반영했습니다~!
`useInput` 활용하는 걸 깜빡했네요 ㅎㅎ 감사합니다!👍